clerk.toml

```diff
@@ -10,67 +10,20 @@ include_dirs = [
   "impot_revenu"
 ]
 
-# -- the following is deprecated, but still used by the LSP --
+[[target]]
+name     = "allocations-familiales"
+modules  = ["Aides_logement"]
+backends = ["ocaml", "python"]
+tests    = ["allocations_familiales/tests/"]
```

Comment:
How do we customize the output directory of this target? Sometimes we don't want it to go in `target/<target-name>` but in another directory.